natashasthilaire/Swampy-Sells
# React Query

### About
This PR builds on Clyde's `useQuery` implementation to attempt to shorten the time it takes for items to be rendered on the homepage. Initially loading the homepage takes a while but after you go to other pages it shouldn't take as long for content to load. Also includes some styling changes for viewing an item.

### Test Instructions
1. Run the `reactquery` test
2. Try going to other routes on the page to see a difference in load time

### Screenshots
![image](https://github.com/natashasthilaire/Swampy-Sells/assets/93110503/a1881669-990a-42c1-9d28-9d1c2ce19047)


### Closes Tickets


### Related Tickets
